Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -982,6 +982,22 @@ void ResourceLoadStatisticsStore::addMissingTablesIfNecessary()
         ITP_RELEASE_LOG_ERROR("addMissingTablesIfNecessary: failed to create unique indices");
 }
 
+
+void ResourceLoadStatisticsStore::addLoginStatusTableIfNecessary()
+{
+    static NeverDestroyed<TableAndIndexPair> missingEntry = std::make_pair<String, std::optional<String>>(createLoginStatus, std::nullopt);
+
+    if (!this->tableExists("LoginStatus"_s)) {
```

Comment:
LoginStatus should probably also be removed from ResourceLoadStatisticsStore::sortedTables and ResourceLoadStatisticsStore::expectedTableAndIndexQueries